myscript.js

```diff
@@ -0,0 +1,195 @@
+
+const resultsURL =  "http://localhost:5050/getFibonacciResults";
+let fibonacciValueURL = "http://localhost:5050/fibonacci/";
+
+
+const fibonacciIndex = document.getElementById("index");
+const fibonacciValue = document.getElementById('y');
+const errorMsg = document.getElementById('myMsg');
+const containerListHolder = document.getElementById('containerDiv');
+const checkBox = document.getElementById('saveCalc');
+const anchorsListDropdown = document.getElementById('aListDropdown');
+const myBtn = document.getElementById('myBtn');
+const spinner = document.getElementById("spinner");
+let isCalculationSaved; 
+let serverMsg = "";
+
+
+
+
+
+
+window.addEventListener('load', FetchAndShowResults);
+
+
+
+
+anchorsListDropdown.addEventListener('click', (e)=> {
+    containerListHolder.innerHTML = "";
+    if (e.target.innerText  === "Number Asc") {   
+        spinnerOn();    
+        FetchAndShowResults("Number Asc");
+    }
+    else if (e.target.innerText  === "Number Desc") {
+        spinnerOn();  
+        FetchAndShowResults("Number Desc");
+    }
+    else if (e.target.innerText  === "Date Asc") {
+        spinnerOn();  
+        FetchAndShowResults("Date Asc");
+    }
+    else if(e.target.innerText  === "Date Desc") {
+        spinnerOn();  
+        FetchAndShowResults("Date Desc");
+    }
+});
+
+
+function spinnerOn() {
+    spinner.classList.add("spinner-border");
+    setTimeout(()=>spinner.classList.remove("spinner-border"),2000);
```

Comment:
Create another function that removes it, who knows how long it takes maybe 0.5 seconds or 1 min